### DISCUSSION TITLE
Is the Foxeer reaper F745 AIO v2 supported?

### DISCUSSION THREAD

**Discussion by Gunsilkyrie** [iNavFlight/inav#8372] [Q&A] [answered]
Hello everyone,

I bought a Foxeer Reaper f745 AIO, but the shop mailed me the Foxeer reaper F745 AIO **_V2_** instead. I guess they have a inventory management issue and they didn't really have it in stock. 
 The Foxeer reaper F745 AIO **_V2_** has the BMI-270 gyro. My question is. Can I use the current Foxeer F745 target for flashing? I just want to make sure I don't break anything :)

**Comment by JulioCesarMatias** [comment, marked as answer]
you can even flash the firmware for your FC, but the Acc and Gyro won't work... But everything else will work fine.

We need to create a sub-target for this FC with the BMI-270 drivers enabled

**Reply by Gunsilkyrie** [reply]
Thank you Julio for replying. Good to know I still will be able to flash it without breaking it. I'll be able to get familiar with INAV. Looking forward to sub-target :)

**Comment by JulioCesarMatias** [comment]
I have a build, you might be interested to know if Acc and Gyro work 
[inav_6.0.0_FOXEERF745AIOV2.zip](https://github.com/iNavFlight/inav/files/9485412/inav_6.0.0_FOXEERF745AIOV2.zip)

For this, iNav Configurator 6.0 will be needed, you can download it here >> http://seyrsnys.myzen.co.uk/inav-configurator-next/

**Reply by Gunsilkyrie** [reply]
Thanks Julio. The Acc and Gyro seems to be working correctly. I don't have a lot of experience with INAV. So I apologize for not able to give a extensive reply :)

**Reply by Surya-deo** [reply]
thanks julio!!, gyro n acc working, baro is not working

**Reply by JulioCesarMatias** [reply]
> thanks julio!!, gyro n acc working, baro is not working

looking at the flight controller specs on the manufacturer's website, it doesn't mention that the target has a Baro
https://www.foxeer.com/foxeer-reaper-aio-v2-f745-45a-bls-bluejay-2-6s-g-439

**Reply by Surya-deo** [reply]
yes, I got, confused.      But what features will be affected by the absence of baro?

**Reply by JulioCesarMatias** [reply]
Using the multirotor profile and no baro... you will not have access to Alt-Hold, Pos-Hold, Return To Home and WayPoint

**Comment by mskfpv** [comment]
Hi all, does iflight blitz f7 aio support inav?
